ql/src/test/org/apache/hadoop/hive/ql/io/parquet/convert/TestETypeConverter.java

```diff
@@ -212,23 +216,59 @@ public void testGetInt64NanosAdjustedToUTCTimestampConverter() throws Exception
   }
 
   @Test
-  public void testGetTextConverter() throws Exception {
+  public void testGetTextConverterForString() throws Exception {
```

Comment:
Nit: the "throws Exception" bit is redundant and can be safely removed

```suggestion
  public void testGetTextConverterForString() {
```